modules/widespaceBidAdapter.js

```diff
@@ -95,10 +95,9 @@ export const spec = {
 
       // Include debug data when available
       if (!isInHostileIframe) {
-        const DEBUG_AD = (find(window.top.location.hash.split('&'),
+        data.forceAdId = (find(window.top.location.hash.split('&'),
```

Comment:
please also fix line 64 above to solve #6465

Reply:
feel free to do this on a different pr

Reply:
So, should I include this code with the new PR, or just fix the bidfloor issue there?

Reply:
your choice